modules/virt-platform-node-storage-requirements.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: REFERENCE
+[id="virt-platform-node-storage-requirements_{context}"]
+= Platform, node, and storage requirements
+
+You can install {VirtProductName} on a cluster that complies with the following platform, node, and storage requirements:
+
+* Platforms:
+
+** On-premise bare metal.
+** Amazon Web Services bare metal instance. Bare metal instances offered by other cloud providers are not supported.
+
+ifdef::openshift-enterprise[]
+:FeatureName: Installing OpenShift Virtualization on an AWS bare metal instance
+include::snippets/technology-preview.adoc[]
+:!FeatureName:
+endif::[]
+
+* Red Hat Enterprise Linux CoreOS (RHCOS) installed on worker nodes. RHEL Linux 7 and 8 nodes are not supported.
+
+* CPUs:
+
+** Supported by link:https://catalog.redhat.com[RHEL 8]. RHEL 9 does not support certain CPUs.
+** Support for Intel 64 or AMD64 CPU extensions.
+** Intel VT or AMD-V hardware virtualization extensions enabled.
+** NX (no execute) flag enabled.
+
+* CSI-enabled cloud storage.
```

Comment:
I am not sure from a technical point of view. Probably the storage also has to be supported by OpenShift. In the best case, we would have a link here.
@aglitke What is your point of view?

Reply:
I will change to OpenShift-supported storage. 

Should CSI be mentioned?